scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/ScalametaTokenizer.scala

```diff
@@ -163,15 +163,9 @@ class ScalametaTokenizer(input: Input, dialect: Dialect) {
       case _ => pushToken(token)
     }
 
-    def loop(braceBalance: Int = 0): Unit = if (nextToken()) {
+    def loop(): Unit = if (nextToken()) {
       emitToken(getCurrToken())
-      lastEmittedToken match {
-        case _: Token.RightBrace if braceBalance > 1 => loop(braceBalance - 1)
-        case _: Token.LeftBrace if braceBalance > 0 => loop(braceBalance + 1)
-        case _: Token.RightBrace => require(braceBalance > 0)
```

Comment:
i think it should be sufficient to remove this and next line. other cases are necessary as splice end can't otherwise be detected.

Reply:
Looks like I actually had to add a case to ignore any additional `}`